design-proposals/high-level-plane.md

```diff
@@ -0,0 +1,57 @@
+# High-level plane
+
+Author: Fabian Deutsch \<fdeutsch@redhat.com\>
+
+## Introduction
+
+Today KubeVirt is exposing a pretty low-level interface to virtualization capabilities.
+
+But this low-level API might be too low-level for i.e. UIs to integrate. On the higher level its typically required to set policies, the lower layer then provides the mechanics to implement this policy. The suggested high-level plane is the middle layer, translating policies into the mechanics.
```

Comment:
I don't like this. Having a VM store sounds ok to me, which has it's own api. The whole high-level thing sounds pretty ominous to me (That does not mean that we can't have a different format on such an endpoint than the low-level VM-spec).

From what we have talked about I though this should just be a an add-on to store VMs while they are not used by the cluster?